jupyter_dashboards/nbextension/notebook/dashboard-view/layout/grid/layout.js

```diff
@@ -627,9 +500,22 @@ define([
             }
             $cell.attr(attrs);
 
+            // add widget to Gridstack, reject if it takes too long
+            var rejectTimeout = window.setTimeout(function() {
+                cellAddedPromise.reject('Timeout waiting for Gridstack to add widget', $cell);
+            }, 10000);
+            var onChange = function(event, elements) {
+                if (elements.some(function(el) { return el.el.is($cell); })) {
```

Comment:
`elements` is an array of objects, each of which contains an `el` jQuery object; notice the contents of the `some` function is `el.el.is()`